### DISCUSSION TITLE
Will drag and drop be added to the tree component in the future

### DISCUSSION THREAD

**Discussion by shiming1113** [quasarframework/quasar#10834] [Ideas / Proposals]
Quasar is cool to use. Recently, we are developing an organization tree function. I want to use the drag and drop function. I see that other UI frameworks have this function (https://2x.antdv.com/components/tree-cn#components-tree-demo-draggable). I hope quasar can consider this function in the future

**Comment by ontwikkelfabriek** [comment]
@shiming1113 

There's an unofficial extension, the repo:
https://github.com/mayank091193/quasar-draggable-tree

The demo:
https://quasar-draggable-tree.netlify.app/examples

> I hope quasar can consider this function in the future

They do
